utils.js

```diff
@@ -102,9 +102,16 @@ async function uploadImageToImgbb(image) {
   }
 }
 
+async function isStaffMember(username) {
+  const staffMembers = process.env.STAFF_MEMBERS.split(",");
```

Comment:
Can you make it get users from the org instead of from the env file. Only include members in the `@is-a-dev/maintainers` team.

Reply:
that would require a user with a pat token @is-a-dev-bot should work however im not sure if it would have to be on the team.

Reply:
@andrewstech The bot is in the organisation so it should be fine.

Reply:
Okay lol need to do this